.github/workflows/release-if-needed.yml

```diff
@@ -92,7 +93,7 @@ jobs:
           cd "${{ needs.check-version.outputs.package_name }}/dist"
           local=$(md5sum ${{ needs.check-version.outputs.dist_package_name }}-${{ needs.check-version.outputs.package_version }}-py3-none-any.whl)
           cd ..
-          pip download ${{ needs.check-version.outputs.package_name }}==${{ needs.check-version.outputs.package_version }} -d . --no-deps --timeout 300
+          pip download ${{ needs.check-version.outputs.dist_package_name }}==${{ needs.check-version.outputs.package_version }} -d . --no-deps --timeout 300
```

Comment:
Is this the fix for the failed package check? If yes, did you test it. It would be good that we don't have it failing yet again on the next release :P

Reply:
I've checked all the names and I hope it should work. The latest problem was that I've used `vizro-core` instead of `vizro`. Now I've added the replacement if `vizro-core` is used.